### DISCUSSION TITLE
Is this pulugin detect windows size?

### DISCUSSION THREAD

**Discussion by Venegrad** [Akryum/v-tooltip#581] [Q&A]
![image](https://user-images.githubusercontent.com/9676703/98588176-ecba2c80-22d3-11eb-96ea-eb0dcc0a0d8a.png)

**Comment by Venegrad** [comment]
![image](https://user-images.githubusercontent.com/9676703/98589230-a665cd00-22d5-11eb-9a8c-f59efbe99efc.png)

**Comment by ghost** [comment]
@Venegrad: https://github.com/Akryum/v-tooltip/issues/144#issuecomment-425366453
